cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppLifecycleHttpHandler.java

```diff
@@ -404,7 +408,7 @@ private void deleteHandler(Id.Program programId, ProgramType type)
           stopProgramIfRunning(programId, type);
           break;
         case WORKFLOW:
-          scheduler.deleteSchedules(programId, ProgramType.WORKFLOW);
+          scheduler.deleteSchedules(programId, SchedulableProgramType.WORKFLOW);
```

Comment:
Currently we only support the scheduling of the Workflows. There is a jira open for supporting scheduling of the other program types (https://issues.cask.co/browse/CDAP-547).